app/models/invite.rb

```diff
@@ -32,7 +32,7 @@ class UserExists < StandardError; end
   def user_doesnt_already_exist
     @email_already_exists = false
     return if email.blank?
-    u = User.find_by("email = ?", Email.downcase(email))
+    u = User.find_by_email(Email.downcase(email))
```

Comment:
Why are we doing `Email.downcase` two times ? (multiple instances of this in the commit)

https://github.com/discourse/discourse/blob/master/app/models/user.rb#L235